pkgs/development/python-modules/Nikola/default.nix

```diff
@@ -37,7 +37,7 @@ buildPythonPackage rec {
   # other hand doesn't support Python 3.3). So, just disable Python 2.
   disabled = !isPy3k;
 
-  buildInputs = [ pytest pytestcov pytest-mock glibcLocales ];
+  checkInputs = [ pytest pytestcov pytest-mock mock glibcLocales ];
```

Comment:
Is `pytest-mock` required by Nikola? I couldn't see it listed in the requirements files nor in setup.py. If not, maybe it could be removed?

Reply:
That's up to you. You're the maintainer :)

Reply:
Oh, true! :D